.evergreen.yml

```diff
@@ -124,6 +124,23 @@ tasks:
 
         COVERALLS_REPO_TOKEN=${coveralls_repo_token|} poetry run coveralls
 
+- name: evg_api_install
+  commands:
+  - command: shell.exec
+    params:
+      working_dir: src
+      script: |
+        set -o errexit
+        set -o verbose
+
+        export LC_ALL=C.UTF-8
+        export LANG=C.UTF-8
+
+        . venv/bin/activate
+
+        poetry install
```

Comment:
I don't think we want to use poetry to test the install. The `poetry install` will set up a dev environment (including installing development dependencies), which we don't expect end-users to use. What we really want to test is installing the package with `pip`.